sdk/attestation/platform-matrix.json

```diff
@@ -0,0 +1,9 @@
+{
+  "displayNames": {
+    "@{ enableVersioning = $true }": ""
+  },
+  "matrix": {
+    "$IMPORT": "eng/pipelines/templates/stages/platform-matrix.json",
+    "ArmTemplateParameters": "@{ enableVersioning = $true }"
```

Comment:
Is there not a good way to set this globally without having to create a wrapper matrix?

Reply:
There could be, but we initially decided to go the import route (remember my ugly `transform: $_ ...` idea? :D ). We could think about command line flags like add/replace/exclude instead of being solely config driven.

Reply:
Lets avoid transforms :) I was just hoping we might have a way to include a global parameter for all matrix entries but this strategy does the job as well.

Reply:
However given you have done this pattern a couple places as well as the MatrixFilter pattern (i.e. not samples or min/max) a couple places it might be worth adding a note or 2 in the JS contributing guidelines that we can point folks to when they want to do this type of thing.

Reply:
Will do. I'm creating an all-up doc, perhaps it will live in tools, that I can link to from all the repos.

Reply:
Yes I'm fine with having a shared doc that can be linked to. However for JS specific patterns they might be better documented in the JS repo.